controllers/movieController.js

```diff
@@ -0,0 +1,41 @@
+const { Movie } = require('../models');
+const { configs } = require('../utils');
+const parser = require('../utils/parser');
+
+/**
+ * Returns all data from every movie.
+ * @return  array of movies
+ */
+module.exports.readAll = async (req, res) => {
+  const qs = await parser.qs(req.url);
+  const allMovies = await Movie.paginate({}, { page: qs.page || 1 });
+
+  res.json(allMovies);
+};
+
+/**
+ * Returns data of specific movie
+ * @param   id  movie id
+ * @return      promise, movie
+ */
+module.exports.readOne = async (req, res) => {
+  const foundMovie = await Movie.findById(req.params.id);
+  res.json(foundMovie);
```

Comment:
Same as above, shouldn't this be an explicit return?
```suggestion
  return res.json(foundMovie);
```

Reply:
Nope.